hexrd/ui/main_window.py

```diff
@@ -217,25 +222,84 @@ def add_materials_panel(self):
                                            self.materials_panel.ui,
                                            'Materials')
 
-    def on_action_open_config_triggered(self):
+    def on_action_open_config_yaml_triggered(self):
+
         selected_file, selected_filter = QFileDialog.getOpenFileName(
             self.ui, 'Load Configuration', HexrdConfig().working_dir,
             'YAML files (*.yml)')
 
         if selected_file:
-            HexrdConfig().working_dir = os.path.dirname(selected_file)
-            HexrdConfig().load_instrument_config(selected_file)
+            path = Path(selected_file)
+            HexrdConfig().working_dir = str(path.parent)
+
+            HexrdConfig().load_instrument_config(str(path))
             self.update_config_gui()
 
-    def on_action_save_config_triggered(self):
+    def on_action_open_config_dir_triggered(self):
+        dialog = QFileDialog(self.ui, 'Save Configuration',
```

Comment:
This should probably be something like `Load Configuration` instead of `Save Configuration` here, right?

Reply:
Good catch!

Reply:
Fixed